data_construction/priorITS_tedersoo/4._format_tedersoo_2014_ITS_prior.r

```diff
@@ -0,0 +1,197 @@
+#prepping tedersoo prior with custom pipeline SV and taxonomy tables.
+#this mostly works, need to do some stuff downstrea of taxonomy table.
+#clear environment, source packages, functions and paths.
+rm(list=ls())
+source('paths.r')
+source('NEFI_functions/fg_assign.r')
+#source('NEFI_functions/lineaus.r')
+source('NEFI_functions/worldclim2_grab.r')
+source('NEFI_functions/arid_extract.r')
+#source('NEFI_functions/crib_fun.r')
+
+#number of genera to keep (top 10 or 20 most abundant)
+n.gen <- 20
+
+#load  files.----
+map <- read.csv(ted_map_raw.path, header = TRUE, na.strings=c("", "NA"))
+map <- data.table(map)
+#load SV table as otu file.
+otu <- readRDS(ted_2014_SV.table.path)
+#load taxonomy.
+tax <- readRDS(ted_2014_tax.path)
+#load times- sent separately by Leho Tedersoo.
+time <- read.csv("/fs/data3/caverill/Microbial_Space_Time_data/tedersoo_2014.data/tedersoo2014_dates.csv", header = TRUE, row.names=1, check.names = FALSE)
```

Comment:
Mentioned this before, but it would be good to not hard-code paths. This is particularly relevant as @zoey-rw starts to get up to speed. I once had two people (a postdoc and lab manager) collectively spend a person year trying to rerun a workflow from a former postdoc and part of what took so long was that all the file paths where hard coded and referenced files in read-only directories on a previous server (UIUC -> BU move)

Reply:
Oof. yes, I agree, all paths need to be in the paths.r folder pointing to somewhere in the larger NEFI_data directory. Just fixed this path. We should talk at some point what to do with raster data (climate data, etc.) Should I make a repo for this in the NEFI data? Or store these somewhere else?  Would be nice if there was a web-based solution to rip points from rasters stored somewhere on the internet.

Reply:
NEFI data repo should have the code for processing data, not the data itself -- for now that can live on geo.

In terms of a web-based solution, some data sources might have APIs and/or R Packages that let us do that. In the more general sense, that should be the sort of thing that Brown Dog would allow us to do, but it might take some tinkering. Let's discuss.